collect_app/src/main/AndroidManifest.xml

```diff
@@ -31,6 +31,11 @@
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+    <!--
```

Comment:
You are right! I've thought that his comment applied to all permissions but after reading thread it's obvious that it might not be the case. I'll correct it.